Tools-Override/tests.targets

```diff
@@ -82,10 +88,13 @@
       <RunTestsForProjectInputs Include="@(_DebugSymbolsIntermediatePath)" />
       <RunTestsForProjectInputs Include="@(AllItemsFullPathWithTargetPath)" />
     </ItemGroup>
-    <ItemGroup>
+    <ItemGroup Condition="'$(_bc_TargetGroup)' != 'netfx'">
       <SupplementalTestData Include="$(XunitRuntimeConfig)" />
       <SupplementalTestData Include="$(RuntimePath)xunit.console.netcore.exe" />
     </ItemGroup>
+    <ItemGroup Condition="'$(_bc_TargetGroup)' == 'netfx'">
+      <SupplementalTestData Include="$(RuntimePath)\*.*" />
```

Comment:
this is needed, otherwise we'll get error 

```
TestExtensions\TestOrderer.cs(19,68): error CS0103: The name 'ReflectionAbstractionExtensions' does not exist in the current context [F:\OSS\corefx\src\System.Linq.Expressions\tests\System.Linq.Expressions.Tests.csproj]
```